<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_client</class>
 <widget class="QDialog" name="gestion_client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tab_client">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>641</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QPushButton" name="ob_tri">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>40</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 0, 0);
background-color: rgb(0, 255, 0);
background-color: rgb(85, 255, 0);</string>
     </property>
     <property name="text">
      <string>TRI</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ob_statistique">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>100</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 0, 0);
background-color: rgb(0, 255, 0);</string>
     </property>
     <property name="text">
      <string>STATISTIQUE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="export2Excel">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>150</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 0, 0);
background-color: rgb(0, 255, 0);</string>
     </property>
     <property name="text">
      <string>EXCEL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ob_cadeaux">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>210</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>255</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>255</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>255</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="NoBrush">
          <color alpha="128">
           <red>85</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>255</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>255</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>255</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="NoBrush">
          <color alpha="128">
           <red>85</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>255</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>255</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>255</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="NoBrush">
          <color alpha="128">
           <red>85</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 85, 0);
background-color: rgb(0, 255, 0);
color: rgb(85, 0, 0);
border-bottom-color: rgb(85, 255, 127);
background-color: rgb(85, 255, 127);</string>
     </property>
     <property name="text">
      <string>CADEAUX</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ob_modifierclient">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>270</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 0, 255);
background-color: rgb(85, 255, 255);</string>
     </property>
     <property name="text">
      <string>MODIFIER CLIENT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic; color:#ff0000;&quot;&gt;NOM :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_name">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 85, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>91</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic; color:#ff0000;&quot;&gt;PRENOM :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_prename">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 85, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>91</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic; color:#ff0000;&quot;&gt;ID :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_Id">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 85, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic; color:#ff0000;&quot;&gt;AGE:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Le_age">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>210</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 85, 255);</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>270</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ob_ajouterclient">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>270</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 255);
background-color: rgb(85, 255, 255);</string>
     </property>
     <property name="text">
      <string>AJOUTER CLIENT</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>ajouter</string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>101</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic; color:#ff0000;&quot;&gt;AGE:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic; color:#ff0000;&quot;&gt;ID:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_id">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 85, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_prenom">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 85, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>91</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic; color:#ff0000;&quot;&gt;PRENOM :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_age">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 85, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic; color:#ff0000;&quot;&gt;NOM :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_nom">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 85, 255);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>afficher</string>
    </attribute>
    <widget class="QTableView" name="tab_client1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>601</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="tri_prenom">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Tri par prenom </string>
     </property>
    </widget>
    <widget class="QRadioButton" name="tri_age">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>83</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Tri par age</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="tri_nom">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>83</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Tri par nom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_9">
    <attribute name="title">
     <string>supprimer</string>
    </attribute>
    <widget class="QLineEdit" name="le_id_supprimer">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>ID : </string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>601</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="trier_nom">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>trier_par_nom </string>
     </property>
    </widget>
    <widget class="QRadioButton" name="trier_prenom">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>-3</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>trier_par_prenom</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="trie_id">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>0</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>trier_par_id</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>rechercher</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>liste des clients </string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Page</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
